### ISSUE TITLE
web: doc comment on WebCommentResponse.code incorrectly says None means error

### ISSUE THREAD

**Issue by dekobon** [dekobon/big-code-analysis#108]
## Summary

The doc comment on `WebCommentResponse.code` states "If `None`, an error occurred processing the request." This is factually incorrect — `None` means no comments were found in the source code, so the original code is unchanged.

## Location

- `big-code-analysis-web/src/web/comment.rs:22-24`

## Evidence

The doc comment:
```rust
/// Source code without comments.
///
/// If `None`, an error occurred processing the request.
pub code: Option<Vec<u8>>,
```

The upstream `rm_comments` function (`src/comment_rm.rs:37-41`) returns `None` when no comments are found:
```rust
if !spans.is_empty() {
    Some(remove_from_code(parser.get_code(), spans))
} else {
    None
}
```

The test at `server.rs:619-623` confirms this:
```rust
// No comment in the code so the code is null
let expected = json!({
    "id": "1234",
    "code": (),
});
```

## Expected Behavior

Doc comment should say: "If `None`, no comments were found in the source code (the original code is unchanged)."

## Actual Behavior

Doc comment incorrectly claims `None` indicates an error.

## Impact

Callers of the API who read the docs may misinterpret a `null` `code` field as an error when it is a normal success case. This leads to incorrect error-handling logic in consumers.

**Comment by dekobon** [comment]
## General Plan for Fix

### Root Cause

The doc comment was written when the semantics of `Option<Vec<u8>>` were assumed to mean error/success, but `rm_comments` uses `None` to mean "no comments to remove."

### Implementation Steps

1. In `big-code-analysis-web/src/web/comment.rs`, change the doc comment on `WebCommentResponse.code` from:
   ```
   /// If `None`, an error occurred processing the request.
   ```
   to:
   ```
   /// If `None`, no comments were found in the source code.
   ```

### Tests to Add/Update

- No test changes needed — this is a doc-only fix.

### Verification

- [ ] `cargo test -p big-code-analysis-web` passes
- [ ] `cargo clippy -p big-code-analysis-web` clean
- [ ] `cargo doc -p big-code-analysis-web --no-deps` builds without warnings

**Comment by dekobon** [comment]
## Fix Summary

Changed the doc comment on `WebCommentResponse.code` in `big-code-analysis-web/src/web/comment.rs:23` from:

> If `None`, an error occurred processing the request.

to:

> If `None`, no comments were found in the source code.

This matches the actual behavior of `rm_comments()` in `src/comment_rm.rs`, which returns `None` when no comments are found (the original code is unchanged).

**Commit:** `cecc611` on branch `fix/issue-108`
